modules/ROOT/pages/apikit-4-forgraphql.adoc

```diff
@@ -0,0 +1,16 @@
+= APIkit for GraphQL Overview
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+GraphQL is a data query and manipulation language that enables API consumers to ask for the data that is only relevant to them. The APIkit for GraphQL module supports the implementation of GraphQL services. These services use connectors to map GraphQL APIs to the existing backends where source data is stored.
```

Comment:
juanchib asked: Worth mention that this service is ideal to use as an experience API in the API-led architecture ?

Reply:
@juanchib, we can say something like:
NOTE: This service is designed to be used as an experience API in the API-led architecture.
Do you think it would be appropriate?

Reply:
Let's leave this change for future discussions.
Thanks for your collaboration!